provisioning/miniprov/hare_mp/main.py

```diff
@@ -201,6 +202,62 @@ def test(args):
         exit(-1)
 
 
+def reset(args):
+    try:
+        rc = 0
+        util: ConsulUtil = ConsulUtil()
+
+        keys = {'epoch': False,
+                'eq-epoch': False,
+                'last_fidk': False,
+                'leader': False,
+                'm0conf/': True,
+                'processes/': True,
+                'stats/': True}
+
+        logging.info('Deleting Hare KV entries (%s)', keys)
+        if not util.kv.kv_delete_in_transaction(keys):
```

Comment:
The function has signature `kv_delete_in_transaction(self, tx_payload: Dict[str, str]) -> bool` while you feed it with `keys` of type `Dict[str, bool]`.

Did you really check this code with mypy? 😉

Reply:
Done